tests/integration/Networking/v2Test.php

```diff
@@ -126,4 +144,130 @@ private function deleteNetwork()
 
         $this->logStep('Deleted network ID', ['ID' => $this->networkId]);
     }
+
+    private function createSubnetsAndDelete()
+    {
+        $replacements = [
+            '{subnetName1}' => 'fakeSubnet1',
```

Comment:
feel free to use [`$this->randomStr()`](https://github.com/php-opencloud/openstack/blob/master/tests/integration/TestCase.php#L96-L107) helper for string generation. It will also add a common prefix, which is useful when teardowns randomly fail to clean up everything